tests/test_suites/ShortSystemTests/test_unlink.sh

```diff
@@ -24,3 +24,9 @@ timeout="3 minutes"
 if ! wait_for '[[ $(find_all_chunks | wc -l) == 0 ]]' "$timeout"; then
 	test_add_failure $'The following chunks were not removed:\n'"$(find_all_chunks)"
 fi
+
+# Ensure thr "unlinked" files are trashed
+trashed_chunk_files=$(find_all_trashed_chunks | wc -l)
+if [ "${trashed_chunk_files}" -eq 0 ]; then
```

Comment:
Please, consider in this test to use probably assert_equals or some similar approach for checking if the number of chunks in trash was the expected one, instead of just checking if there is some trash or not.

Reply:
Yes, I consider it good now. Just take into account for comparing expected values with some variable value or function output to follow the conventions on other tests by performing this comparisons by using `assert_equals` or `assert_not_equal` . This is a minor change as it is most for having same coding style across all integration tests.